tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/JmapSettingsGetMethodContract.scala

```diff
@@ -0,0 +1,291 @@
+           |        "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+           |        "state": "${JmapSettingsStateFactory.INITIAL.serialize}",
+           |        "list": [],
+           |        "notFound": [
+           |            "notFound1"
+           |        ]
+           |    },
+           |    "c1"
+           |]""".stripMargin))
+
+
+  @Test
+  def getShouldReturnValidResponseWhenSingletonId(): Unit =
+    `given`
+      .body(
+        s"""{
+           |	"using": ["urn:ietf:params:jmap:core", "com:linagora:params:jmap:settings"],
+           |	"methodCalls": [
+           |		[
+           |			"Settings/get",
+           |			{
+           |				"accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+           |				"ids": ["singleton"]
+           |			},
+           |			"c1"
+           |		]
+           |	]
+           |}""".stripMargin)
+      .when
+      .post.prettyPeek()
+      .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .body("methodResponses[0]", jsonEquals(
+        s"""[
+           |    "Settings/get",
+           |    {
+           |        "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+           |        "state": "${JmapSettingsStateFactory.INITIAL.serialize}",
+           |        "list": [{
+           |                    "id": "singleton",
+           |                    "settings": {}
+           |                }],
+           |        "notFound": []
+           |    },
+           |    "c1"
+           |]""".stripMargin))
+
+
+  @Test
+  def mixedFoundAndNotFoundCase(server: GuiceJamesServer): Unit = {
+    val settingsStateUpdate = server.getProbe(classOf[JmapSettingsProbe])
+      .reset(BOB, Map(("key1", "value1")))
+
+    `given`
+      .body(
+        s"""{
+           |	"using": ["urn:ietf:params:jmap:core", "com:linagora:params:jmap:settings"],
+           |	"methodCalls": [
+           |		[
+           |			"Settings/get",
+           |			{
+           |				"accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+           |				"ids": ["singleton", "notFound2"]
+           |			},
+           |			"c1"
+           |		]
+           |	]
+           |}""".stripMargin)
+      .when
+      .post.prettyPeek()
+      .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .body("methodResponses[0]", jsonEquals(
+        s"""[
+           |    "Settings/get",
+           |    {
+           |        "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+           |        "state": "${settingsStateUpdate.newState.serialize}",
+           |        "list": [{
+           |                    "id": "singleton",
+           |                    "settings": {"key1": "value1"}
+           |                }],
+           |        "notFound": ["notFound2"]
+           |    },
+           |    "c1"
+           |]""".stripMargin))
+  }
```

Comment:
Should we have a test that makes sure delegatees can use `Settings/get` on behalf of the delegator?

Reply:
Wrong accountId as well?